did-you-know/src/plugin.ts

```diff
@@ -61,7 +61,7 @@ export default (api: any) => {
     // 1、get matched
     const matched = items.filter((item: any) => {
```

Comment:
```suggestion
    const matched = items.filter(item => {
```

应该可以去掉 `any` 了，这个坑竟然现在才发现...